src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/PaymentOrder/Update/UpdateTests.cs

```diff
@@ -35,7 +36,7 @@ public async Task Update_PaymentOrder(Product[] products, PayexInfo payexInfo)
                 .Actions.Rows[y => y.Name.Value.Contains(PaymentOrderResourceOperations.PaidPaymentOrder)].Should.BeVisible()
                 .Actions.Rows.Count.Should.Equal(3);
 
-            var order = await SwedbankPayClient.PaymentOrders.Get(orderLink, SwedbankPay.Sdk.PaymentOrders.PaymentOrderExpand.All);
+            var order = await SwedbankPayClient.PaymentOrders.Get(orderLink, PaymentOrderExpand.All);

```

Comment:
Codacy found an issue: [Remove the unused local variable 'order'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)